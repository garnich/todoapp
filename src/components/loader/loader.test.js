import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Loader from './loader'

Enzyme.configure({ adapter: new Adapter() })

describe('Test Loader component', () => {
  it('<Loader>', () => {
    const wrapper = shallow(<Loader />)

    expect(wrapper).toMatchSnapshot()
  })
})
