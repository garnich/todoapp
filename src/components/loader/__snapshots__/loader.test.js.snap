// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loader component <Loader> 1`] = `
<div
  className="loader"
>
  <div
    className="spinner"
  />
</div>
`;
